packages/ng/libraries/core/src/lib/overlay/sidepanel/sidepanel-panel.component.ts

```diff
@@ -10,6 +10,7 @@ import { ALuModalPanelComponent } from '../modal/index';
 	selector: 'lu-sidepanel-panel',
 	templateUrl: './sidepanel-panel.component.html',
 	styleUrls: ['./sidepanel-panel.component.scss'],
+	host: {'class': 'lu-sidepanel-panel'},

```

Comment:
i think the `host` atribute is deprecated and supposed to disappear in favor of `@HostBinding` decorator